ragstack-e2e-tests/pyproject.langchain.toml

```diff
@@ -21,7 +21,7 @@ beautifulsoup4 = "^4"
 
 langchain = { git = "https://github.com/langchain-ai/langchain.git", branch = "master", subdirectory = "libs/langchain", extras = ["openai"] }
 langchain_core = { git = "https://github.com/langchain-ai/langchain.git", branch = "master", subdirectory = "libs/core" }
-llama-index = { version = "0.9.11", extras = ["langchain"] }
+llama-index = { version = "v0.9.14", extras = ["langchain"] }
```

Comment:
```suggestion
llama-index = { version = "0.9.14", extras = ["langchain"] }
```

Reply:
fixed, thanks.
I don't know why it worked locally for me